.github/stale.yml

```diff
@@ -0,0 +1,63 @@
+# Configuration for probot-stale - https://github.com/probot/stale
+
+# Number of days of inactivity before an Issue or Pull Request becomes stale
+daysUntilStale: 90
```

Comment:
I've never actually worked extensively on a project with a stale bot, but this seems short to me. 

To put this into concrete terms:
- Only the most recent 15 of the currently open 62 issues were opened less than 90 days ago. I'm not sure how to easily tell how many of the remaining issues would become stale.
- For an issue that one person wanted to keep open, they'd have to comment 4 times a year to prevent it from being closed, unless we add a label to keep it open forever.

If we did something like 365 days, we'd instead end up with just over half the issues potentially being flagged—27/62 were opened in the last year. The upside is it gives issues more breathing room for things people care about without them feeling like they're nagging the maintainers. 

I'm not attached to any number in particular, and we can always change it, but it seems like starting conservatively and shortening if issues are piling up too quickly would be safer than starting with a small window.

Reply:
@rliebz

> I've never actually worked extensively on a project with a stale bot

Everyone has different experiences ✨ this is the first project I've worked _(for longer than a few weeks)_ on that doesn't have "stale" handling of some sort.

> Only the most recent 15 of the currently open 62 issues were opened less than 90 days ago.

The stale bot uses the time of the issue's most recent **update**, not the **creation time**. More details here => https://github.com/probot/stale#how-are-issues-and-pull-requests-considered-stale

_(the rest of your comment likely needs to be reconsidered in light of this new information, so I'm going to skip replying to it for the moment)_